src/OSSupport/Socket.cpp

```diff
@@ -1,377 +0,0 @@
-
```

Comment:
It is. The files are not used by any project, and since the stylechecking CI doesn't use cmake to produce `AllFiles.lst`, and rather scans all .cpp and .h files, this was in the way.